var/spack/repos/builtin/packages/warpx/package.py

```diff
@@ -84,7 +84,7 @@ class Warpx(CMakePackage):
     depends_on('fftw +mpi', when='+psatd +mpi compute=omp')
     depends_on('lapackpp', when='+psatd dims=rz')
     depends_on('mpi', when='+mpi')
-    depends_on('openpmd-api@0.13.1:,dev', when='+openpmd')
+    depends_on('openpmd-api@0.13.1:,develop', when='+openpmd')
```

Comment:
I can probably even remove this now, do I? :)
```suggestion
    depends_on('openpmd-api@0.13.1:', when='+openpmd')
```